client/my-sites/plan-features-2023-grid/components/test/billing-timeframe.tsx

```diff
@@ -122,6 +124,105 @@ describe( 'PlanFeatures2023GridBillingTimeframe', () => {
 		const discountedPrice = formatCurrency( planPrices.discountedPrice, 'INR', {
 			stripZeros: true,
 		} );
-		expect( container ).toHaveTextContent( `per month, ${ discountedPrice } for the first year` );
+		expect( container ).toHaveTextContent(
+			`per month, ${ discountedPrice } for the first two years`
+		);
+	} );
+
+	test( 'should show full-term discounted price when plan is 3-yearly', () => {
+		const planPrices: PlanPricesDisplay = {
+			discountedPrice: 150,
+			originalPrice: 200,
+		};
+
+		usePlanPricesDisplay.mockImplementation( jest.fn( () => planPrices ) );
```

Comment:
`Non-blocking / Nit`:

Not necessary to do in this PR. I see we have a `beforeEach`, but how would we feel about an `afterAll` to to tear down mocks?

```javascript
afterAll(() => {
  jest.clearAllMocks();
});
```